### ISSUE TITLE
Lint fails for newly generated project

### ISSUE THREAD

**Issue by karpikpl** [kriasoft/babel-starter-kit#2]
Hi,
I've run `yo javascript` and then `npm run lint`, the result:

```
$ npm run lint

> functional-js@0.0.0 lint c:\Projects\github\functionalJs
> eslint src docs/js tools


tools\utils\compile.js
  35:18  error  Empty block statement  no-empty
  40:16  error  Empty block statement  no-empty

✖ 2 problems (2 errors, 0 warnings)


npm ERR! Windows_NT 6.3.9600
npm ERR! argv "c:\\Program Files\\nodejs\\node.exe" "c:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "lint"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4
npm ERR! code ELIFECYCLE
npm ERR! functional-js@0.0.0 lint: `eslint src docs/js tools`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the functional-js@0.0.0 lint script 'eslint src docs/js tools'.
npm ERR! This is most likely a problem with the functional-js package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     eslint src docs/js tools
npm ERR! You can get their info via:
npm ERR!     npm owner ls functional-js
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\Projects\github\functionalJs\npm-debug.log
```

Is that by design or can be fixed?

**Comment by koistya** [comment]
@karpikpl you may want to run the local version of ESLint `npm run lint` (see `package.json/scripts/lint`).

**Comment by koistya** [comment]
Fixed in `v1.0.0`
